doc/crypto/api/ops/signature.rst

```diff
@@ -856,6 +1107,22 @@ Support macros
                          hash, sizeof(hash), &hash_len);
         psa_sign_hash(key, alg, hash, hash_len, ...);
 
+.. macro:: PSA_ALG_SUPPORTS_CONTEXT
```

Comment:
Wondering about other possible naming options for this macro: `PSA_ALG_USES_CONTEXT()`...?

Reply:
I wondered about that, but settled on Supports. 
I have clarified the wording to note that it reflects the implementation of the algorithm.